src/solver.rs

```diff
@@ -230,6 +398,10 @@ pub fn solve_equation(input: &str, word_list: &[&str], num_results_requested: us
     //    This holds each pattern string, its parsed form, and its `lookup_keys` (shared vars).
     let patterns = Patterns::of(input);
 
+    // 1.5 Build per-pattern lookup key specs (shared vars) for the join
```

Comment:
is this step numbered 1.5 b/c it's particularly subordinate (or the like), or is it just not yet updated? (or something else?)

Reply:
I'm changing the numbering; we needed it anyway